README.md

```diff
@@ -273,6 +279,22 @@ left untouched!",
 
         // inline arrays are just arrays with the correct type ...
         "inline_array_member1"  : [1, 2, 3],
+		
+		// an anyptr can be null, and then it is typeless
```

Comment:
I'll complain about indent here as this will not be fixable by a clang-format :)